### ISSUE TITLE
Accessibility name and description are missing  'with=accessibility'

### ISSUE THREAD

**Issue by olli-suutari-jkl** [libraries-fi/kirkanta-api#1]
Queries with: 'with=accessibility' only return 
'      "meta": {
        "created": "2015-12-01T13:34:11+00:00",
        "modified": "2015-12-01T15:34:11+02:00"
      }
' 
for each created accessibility instead of name/description.

https://api.kirjastot.fi/v3/library/85754?lang=fi&with=accessibility

**Comment by sjuvonen** [comment]
There has never really been an implementation for accessibility data in Kirkanta. An older version had a couple of checkboxes that were pretty much ignored by libraries, so we ended up dropping it altogether. 

We are now investigating how we could introduce accessibility data in a sensible manner, preferably so that we can utilize data from palvelukartta.hel.fi and suomi.fi/PTV.

**Comment by olli-suutari-jkl** [comment]
Since the new "Esteettömyyspalvelut | Saavutettavuus | Palvelu" -service was added in V4, this is no longer an issue.
